api/src/main/resources/messages.properties

```diff
@@ -1667,6 +1667,7 @@ Patient.merge.notPreferred=Not Preferred
 Patient.merge.includeVoided=Include deleted
 Patient.merged=Patients merged successfully
 Patient.merge.patient.NoUnvoidedOrders=Not preferred patient(s) should not have any unvoided orders
+Patient.merge.bothHasActiveOrders=Cannot merge patients. Both {0} and {1} patients have active [{2}] orders.
```

Comment:
Lets go with this: Patient.merge.bothHasSameTypeActiveOrders.
What do you think?